.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: ci
 # yamllint disable-line rule:truthy
-on: [push, pull_request]
+on:
+  pull_request:
```

Comment:
IMO this should also have master branch. is there any reason to run the CI for PR on branches other than the master?

Reply:
why would you want to exclude non-master branches? If there is for example a "sle12" branch, we should also run tests for that branch

Reply:
i see your point. My question is why then you need ci on push only for master then? i think i miss something!

Reply:
seems you had similar question yesterday and there were some answers. i will resolve this conversation.